fluentlenium-core/src/main/java/org/fluentlenium/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,105 @@
+package org.fluentlenium.utils;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Utility class for reflection.
+ */
+public abstract class ReflectionUtils {
+    private static Class<?>[] EMPTY_CLASS_ARRAY = new Class<?>[0];
+
+    @SuppressWarnings("unchecked")
+    public static <T> Class<T> wrapPrimitive(Class<T> c) {
+        return c.isPrimitive() ? (Class<T>) PRIMITIVES_TO_WRAPPERS.get(c) : c;
+    }
+
+    private static final Map<Class<?>, Class<?>> PRIMITIVES_TO_WRAPPERS = new HashMap<>();
+
+    static {
+        PRIMITIVES_TO_WRAPPERS.put(boolean.class, Boolean.class);
+        PRIMITIVES_TO_WRAPPERS.put(byte.class, Byte.class);
+        PRIMITIVES_TO_WRAPPERS.put(char.class, Character.class);
+        PRIMITIVES_TO_WRAPPERS.put(double.class, Double.class);
+        PRIMITIVES_TO_WRAPPERS.put(float.class, Float.class);
+        PRIMITIVES_TO_WRAPPERS.put(int.class, Integer.class);
+        PRIMITIVES_TO_WRAPPERS.put(long.class, Long.class);
+        PRIMITIVES_TO_WRAPPERS.put(short.class, Short.class);
+        PRIMITIVES_TO_WRAPPERS.put(void.class, Void.class);
+    }
+
+
+    /**
+     * <p>Converts an array of {@code Object} in to an array of {@code Class} objects.
+     * If any of these objects is null, a null element will be inserted into the array.</p>
+     * <p>
+     * <p>This method returns {@code null} for a {@code null} input array.</p>
+     *
+     * @param array an {@code Object} array
+     * @return a {@code Class} array, {@code null} if null array input
+     */
+    public static Class<?>[] toClass(final Object... array) {
+        if (array == null) {
+            return null;
+        } else if (array.length == 0) {
+            return EMPTY_CLASS_ARRAY;
+        }
+        final Class<?>[] classes = new Class[array.length];
+        for (int i = 0; i < array.length; i++) {
+            classes[i] = array[i] == null ? null : array[i].getClass();
```

Comment:
For me it would be easier to read if you just add parenthesis.

`
classes[i] = (array[i] == null) ? null : array[i].getClass();
`